frontend/institution/institutionController.js

```diff
@@ -263,6 +269,20 @@
             });
         };
 
+        institutionCtrl.editDescription = function(ev) {
+            $mdDialog.show({
+                templateUrl: 'app/institution/descriptionInst/edit_description.html',
+                targetEvent: ev,
+                clickOutsideToClose:true,
+                locals: {
+                    institution: institutionCtrl.institution,
```

Comment:
why do you need both institution and institution key as parameters? Doenst institution object have it own key property?